app/views/catalog/_constraints.html.erb

```diff
@@ -2,7 +2,7 @@
 
       <div id="appliedParams" class="clearfix constraints-container">
         <div class="pull-right">
-          <%=link_to t('blacklight.search.start_over'), start_over_path, :class => "catalog_startOverLink btn btn-sm btn-text", :id=>"startOverLink" %>
+          <%=link_to t('blacklight.search.start_over'), advanced_path, :class => "catalog_startOverLink btn btn-sm btn-text", :id=>"startOverLink" %>
```

Comment:
Yeah, [Blacklight's CSS is full of it](https://github.com/projectblacklight/blacklight/blob/6e9151de47e74a1dc8efea7d2e87effa02b8a816/app/assets/stylesheets/blacklight/_catalog.css.scss). Some kind of weird SMACSS-style semantic thing I assume